src/components/FooterBrick/index.js

```diff
@@ -1,111 +1,100 @@
-import React from 'react';
-import styles from './styles';
-import { ThemePropagator, CallToAction } from '../../';
+import React, { Component } from 'react';
 import FontAwesome from 'react-fontawesome';
+import styles from './styles';
 
-function renderIcon(icon, s) {
-  if (!icon) return;
-  return (<FontAwesome style={ s.icon } name={icon} />);
-}
+class FooterBrick extends Component {
 
-function renderItems(item, s) {
-  if (item.icon) return <li key={ item.key } >{ renderIcon(item.icon, s) }<a style={ s.text } target="_blank" href={ item.link } >{ item.text }</a></li>
-  if (item.link) return <li key={ item.key } ><a style={ s.text } target="_blank" href={ item.link } >{ item.text }</a></li>
-  if (item) return <li key={ item.key } ><a style={ s.text }>{ item.text }</a></li>
-  return null;
-};
+  renderSimpleFooter(s) {
+    let { title, description, image } = this.props;
 
-function renderColumn1(props, s) {
-  if (!props.itemsColumn1) return (<div style={ s.textColumn2 }></div>);
-  return(
-    <div style={ s.textColumn1 }>
-      <ul style={ s.noList }>
-        { props.itemsColumn1.map((items) => renderItems(items, s)) }
-      </ul>
-    </div>
-  );
-};
-
-function renderColumn2(props, s) {
-  if (!props.itemsColumn2) return (<div style={ s.textColumn2 }></div>);
-  return(
-    <div style={ s.textColumn2 }>
-      <ul style={ s.noList }>
-        { props.itemsColumn2.map((items) => renderItems(items, s)) }
-      </ul>
-    </div>
-  );
-};
+    return(
+      <div style={ s.center }>
+        { (title) ? <h1>{ title }</h1> : null }
+        { (description) ? <p>{ description }</p> : null }
+        { (image) ? <img src={ image } height="50" alt="logo" /> : null }
+      </div>
+    )
+  }
 
-function renderColumn3(props, s) {
-  if (!props.itemsColumn3) return (<div style={ s.textColumn2 }></div>);
-  return(
-    <div style={ s.textColumn3 }>
-      <ul style={ s.noList }>
-        { props.itemsColumn3.map((items) => renderItems(items, s)) }
-      </ul>
-    </div>
-  );
-};
+  renderExpandFooter(s) {
```

Comment:
renderExpandFooter is the only way in which we are going to render, so it should be called "render" and nothing else.